src/tests/integration/test-data/show/AddConst.applyOp(uint256).cse.expected

```diff
@@ -1,52 +1,59 @@
 
-┌─ 1 (root, split, init)
+┌─ 1 (root, init)
```

Comment:
The `split` should remain, it not being here indicates that the proof has not been minimized.

Reply:
It's no longer splitting.

Reply:
Something feels not right here, this and other CSE proofs of non-tests should be splitting at the top node.